beast/tools/star_type_probability.py

```diff
@@ -0,0 +1,244 @@
+import numpy as np
+from collections import defaultdict
+
+from astropy.table import Table
+from astropy.io import fits
+
+
+def star_type_probability(
+    pdf1d_files,
+    pdf2d_files,
+    output_filebase=None,
+    ext_O_star_params=None,
+    dusty_agb_params=None,
+):
+    """
```

Comment:
Need a short description of what this function is doing.

Reply:
Standard part of docstrings.

Reply:
Thanks for catching that!